securitas/templates/user-edit.html

```diff
@@ -22,6 +22,18 @@
           </ul>
           <div class="tab-content">
             <div class="tab-pane fade show active" id="settings-personal" role="tabpanel">
+              <div class="card-body pb-0">
+                <label>Avatar</label>
```

Comment:
I think the formatting here is a bit funny now. We could probably just remove the word avatar and add it to alt of the img just in case. Also could we put the change avatar link below the actual avatar?